news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 40 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   XXXX
+authors: [ PrestaShop ]
```

Comment:
You can modify the author 😉

Reply:
it's ok like that. after all, if we would count the pull request on Build too, I would be credited for those core-weekly.